client/my-sites/customer-home/main.jsx

```diff
@@ -84,25 +114,35 @@ class Home extends Component {
 			);
 		}
 
-		const { siteId, hasChecklistData, isChecklistComplete } = this.props;
+		const { siteId, hasChecklistData, isChecklistComplete, checklistMode } = this.props;
+		const renderChecklistCompleteBanner = 'render' === this.state.renderChecklistCompleteBanner;
 
 		return (
 			<Main className="customer-home__main is-wide-layout">
 				<PageViewTracker path={ `/home/:site` } title={ translate( 'Customer Home' ) } />
 				<DocumentHead title={ translate( 'Customer Home' ) } />
 				<SidebarNavigation />
-				{ siteId && ! hasChecklistData && <QuerySiteChecklist siteId={ siteId } /> }
-				{ hasChecklistData && (
-					<Fragment>
-						{ isChecklistComplete ? this.renderCustomerHome() : this.renderChecklist() }
-					</Fragment>
+				{ renderChecklistCompleteBanner && (
+					<Banner
+						dismissPreferenceName
```

Comment:
I was getting another warning about this not being the right type. I changed it to:

```
dismissPreperenceName="checklist-complete"
```

and it seemed satisfied.

Reply:
Cool, will fix